lib/_http_client.js

```diff
@@ -40,7 +39,7 @@ const Agent = require('_http_agent');
 const { Buffer } = require('buffer');
 const { defaultTriggerAsyncIdScope } = require('internal/async_hooks');
 const { URL, urlToOptions, searchParamsSymbol } = require('internal/url');
-const { outHeadersKey, ondrain } = require('internal/http');
+const { outHeadersKey, needDrainKey } = require('internal/http');
```

Comment:
I would stick to the naming you had before (`kNeedDrain`) so that it's easy to tell that it's a symbol key and not any other kind of variable.

Reply:
Ok. Should I change outheaderskey as well?

Reply:
That's probably better for a separate PR.

Reply:
Fixed

Reply:
https://github.com/nodejs/node/pull/29091